tox.ini

```diff
@@ -2,8 +2,8 @@
 min_version = 4.11
 
 env_list =
-    python{39}-django4.2-wagtail{6.3,6.4,7.0}
-    python{310,311,312,313}-django{5.1,5.2}-wagtail{6.3,6.4,7.0,main}
+    python{39}-django4.2-wagtail{6.3,7.0,7.1}
+    python{310,311,312,313}-django{5.1,5.2}-wagtail{6.3,7.0,7.1,main}
```

Comment:
as with wagtailmedia, this now has way too many combinations that can be reduced a bit.